src/generator_p.h

```diff
@@ -80,30 +80,27 @@ struct Generator::GeneratorImpl
 
     void addStateAndVariableCountCode(bool interface = false);
 
-    void addVariableTypeObjectCode();
-
     std::string generateVariableInfoObjectCode(const std::string &objectString) const;
 
     void addVariableInfoObjectCode();
 
     std::string generateVariableInfoEntryCode(const std::string &name,
                                               const std::string &units,
-                                              const std::string &component,
-                                              const std::string &type) const;
+                                              const std::string &component) const;
+
+    void addInterfaceVariableInfoCode();
 
-    void addInterfaceVoiStateAndVariableInfoCode();
-    void addImplementationVoiInfoCode();
-    void addImplementationStateInfoCode();
+    void doAddImplementationVariableInfoCode(const std::string &variableInfoString,
```

Comment:
Using ‘do’ as a prefix has been used in this codebase to indicate that this is the implementation via a virtual function. Keeping virtual methods out of the API is better for binary stability. This function is not virtual and I think should make use of C++ override functionality instead.